.github/workflows/workflow.yml

```diff
@@ -80,6 +80,7 @@ jobs:
           sudo apt-get update --fix-missing
           sudo apt-get autoremove
           sudo apt-get autoclean
+          pip install --user --upgrade setuptools
```

Comment:
these are due to the following issue, after having added dependencies of the form `name @ uri`:

```
error in aea setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
```

Reply:
let's discuss

Reply:
will be removed

Reply:
done